src/server/scripts/Northrend/Gundrak/boss_moorabi.cpp

```diff
@@ -107,6 +107,7 @@ class boss_moorabi : public CreatureScript
 
             void EnterEvadeMode(EvadeReason /*why*/) override
             {
+                me->RemoveAurasOnEvade();
```

Comment:
wait a second...
It should happen by default
Is a new bug or this auras should persist after evade/death?